web/src/components/CreateAssertionModal/CreateAssertionModalProvider.tsx

```diff
@@ -0,0 +1,58 @@
+import {noop} from 'lodash';
```

Comment:
New provider for the modal, it allows us to keep a single instance for the whole page and easy to use with the context hook